vclib-openid/src/commonTest/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopIsoProtocolTest.kt

```diff
@@ -154,14 +147,16 @@ class OidcSiopIsoProtocolTest : FreeSpec({
             encryption = true
         )
         val authnRequest = verifierSiop.createAuthnRequestUrl(
-            walletUrl = walletUrl,
-            requestOptions = requestOptions
+            walletUrl = walletUrl, requestOptions = requestOptions
```

Comment:
Please keep the line breaks

Reply:
fixed in commit 4202e428c379ed6cb86cd537e498bf43e68f5294